test/tests/jobs/CancelJobTest.cpp

```diff
@@ -313,24 +313,47 @@ struct CancelJobTest : tpunit::TestFixture {
         command["name"] = "parent";
         tester->executeWaitVerifyContent(command);
 
-        // Create the child
+        // Create one child
         command.clear();
         command.methodLine = "CreateJob";
         command["name"] = "child";
         command["parentJobID"] = parentID;
         response = tester->executeWaitVerifyContentTable(command);
         string childID = response["jobID"];
 
+        // Create another child
+        command.clear();
+        command.methodLine = "CreateJob";
+        command["name"] = "child2";
+        command["parentJobID"] = parentID;
+        response = tester->executeWaitVerifyContentTable(command);
+        string childID2 = response["jobID"];
+
         // Finish the parent to put the child in the QUEUED state
         command.clear();
         command.methodLine = "FinishJob";
         command["jobID"] = parentID;
         tester->executeWaitVerifyContent(command);
 
-        // Try cancelling one of the children, but none of them are running so it should fail.
+        // Cancel one child
         command.clear();
         command.methodLine = "CancelJob";
         command["jobID"] = childID;
-        tester->executeWaitVerifyContent(command, "404 Invalid jobID - Cannot cancel a job that has no RUNNING/RUNQUEUED siblings");
+        tester->executeWaitVerifyContent(command);
```

Comment:
Can you check the state of the child jobs and confirm that they are `CANCELLED`?